sdf_timing/sdflex.py

```diff
@@ -150,7 +150,7 @@ def t_SLASH(t):
 
 
 def t_STRING(t):
-    r'[a-zA-Z0-9_\/.\[\]]+'
+    r'[a-zA-Z0-9_\/.\[\]\\\\]+'
```

Comment:
I believe this can be `\\` (not `\\\\`)

Reply:
Fixed by commit 3513c57. Not sure why I thought I needed 4, seemed to remember it did not work with 2 only.